lib/streamlit/commands/page_config.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2018-2020 Streamlit Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from streamlit.delta_generator import _enqueue_message
+from streamlit.proto import ForwardMsg_pb2
+from streamlit.proto import PageConfig_pb2
+from streamlit.elements import image_proto
+from streamlit.errors import StreamlitAPIException
+
+
+def set_page_config(
+    page_title=None, page_icon=None, layout="centered", initial_sidebar_state="auto"
+):
+    """
+    Configures the default settings of the page.
+
+    .. note::
+        This must be the first Streamlit command used in your app, and must only
+        be set once.
+
+        This is a beta feature. See
+        https://docs.streamlit.io/en/latest/pre_release_features.html for more
+        information.
+
+    Parameters
+    ----------
+    page_title: str or None
+        The page title, shown in the browser tab.
+        Strings get appended with "• Streamlit".
+    page_icon : Anything supported by st.image or str or None
+        The page favicon.
+        Besides the types supported by `st.image` (like URLs or numpy arrays),
+        you can pass in an emoji as a string ("🦈") or a shortcode (":shark:").
+        Emoji icons are courtesy of Twemoji and loaded from MaxCDN.
+    layout: "centered" or "wide"
+        How the page content should be laid out.
+        "centered" constrains the elements into a centered column of fixed width;
+        "wide" uses the entire screen.
+    initial_sidebar_state: "auto" or "expanded" or "collapsed"
+        How the sidebar should start out.
+        "auto" hides the sidebar on mobile-sized devices, and shows on desktop.
+        "expanded" shows the sidebar initially; "collapsed" hides it.
```

Comment:
So we don't do this consistently and though this is redundant, I personally like having it documented what the default value is for a param that doesn't default to `None`, i.e. `layout` and `initial_sidebar_state`.

Reply:
Good idea, done. Also mentioned that the default page title for "app.py" is "app * Streamlit"